services/QuillLMS/spec/requests/secure_headers_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+class DummyController < ApplicationController
+  def index 
+    cookies['foo'] = 'bar'
+    render json: {}
+  end
+end
+
+describe DummyController, type: :request do
+  before do 
+    Rails.application.routes.draw do
+      get '/dummy', to: 'dummy#index'
+    end
+  end 
+
+  after do 
+    Rails.application.reload_routes!
+  end
+
+  it 'should flag all cookies as HttpOnly' do 
+    get '/dummy'
+    expect(response.header['Cache-Control']).to match('no-cache, no-store, max-age=0, must-revalidate')
+    expect(response.header['Pragma']).to match('no-cache')
```

Comment:
Note: I'm not testing the Expires header here because it does not do anything. I'm including it in the code for CB compliance only.

From Mozilla:
"If there is a Cache-Control header with the max-age or s-maxage directive in the response, the Expires header is ignored."